### ISSUE TITLE
Stdout used instead of stderr

### ISSUE THREAD

**Issue by iguerNL** [OCamlPro/alt-ergo#393]
- This behavior is noticed for models (which can be on stdout in the default setting, but we should be able to select a different std)

- The behavior is also noticed on Alt-Ergo's output on SMT files:
```
./alt-ergo foo.smt2 2> /dev/null 
; File "foo.smt2", line 1, characters 1-12: I don't know (0.0021) (0 steps) (goal g_1)
unknown
```
The line `; File "foo.smt2", line 1, characters 1-12: I don't know (0.0021) (0 steps) (goal g_1)` above should be sent to stderr

**Comment by ACoquereau** [comment]
Should be fixed with #408  and #417

**Comment by bclement-ocp** [comment]
Models are on stdout as per smtlib recommendations; the second issue is fixed. Closing.

We should consider printing the models from `--dump-models` on stderr (or have a configuration flag for those).
